### PR TITLE
[lunastreaming] Add experimental distribution info

### PR DESCRIPTION
**Pull request by nicoloboschi**

### CODE REVIEW

**Review by cdbartholomew — APPROVED** [review]
LGTM

File: Luna_Streaming_2.10_Release_Notes.md [hunk 1]
```diff
@@ -76,6 +76,8 @@ The distributions are available as Docker images and tarballs, and both methods
 * **lunastreaming**: the basic Luna Streaming Distribution **including Pulsar SQL** feature.
 * **lunastreaming-core**: the basic Luna Streaming Distribution **without Pulsar SQL** feature. Pick this distribution if you don't need Pulsar SQL features.
 * **lunastreaming-all**: contains the Core Luna Streaming Distribution, including **Pulsar Offloaders** and the **Datastax Pulsar IO Connectors** listed above. Pick this distribution if you want the Datastax connectors or the offloading feature.
+* **lunastreaming-experimental**: extends the **all** distribution with experimental features and connectors. You can find a detailed list experimental addition [here](Luna_Streaming_Experimental.md). This distribution is only available as a Docker image.
```
Comment (mendonk) [inline Luna_Streaming_2.10_Release_Notes.md:79]:
```suggestion
* **lunastreaming-experimental**: extends the **all** distribution with experimental features and connectors. You can find a detailed list of the experimental additions [here](Luna_Streaming_Experimental.md). This distribution is only available as a Docker image.
```

File: Luna_Streaming_Experimental.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# DataStax Luna Streaming Experimental Distribution
+
+In the **experimental** distribution several connectors are added to the **all** distribution but they are NOT officialy supported and NOT production ready.
+
+## Builtin Apache Pulsar connectors
+
+These connectors are mantained in the Apache Pulsar community. You can follow the [official guide](https://pulsar.apache.org/docs/io-connectors) to learn how to use them.
```
Comment (mendonk) [inline Luna_Streaming_Experimental.md:7]:
```suggestion
These connectors are maintained by the Apache Pulsar community. You can follow the [official guide](https://pulsar.apache.org/docs/io-connectors) to learn how to use them.
```

File: Luna_Streaming_Experimental.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+* pulsar-io-netty-VERSION.nar
+* pulsar-io-nsq-VERSION.nar
+* pulsar-io-rabbitmq-VERSION.nar
+* pulsar-io-redis-VERSION.nar
+* pulsar-io-solr-VERSION.nar
+* pulsar-io-twitter-VERSION.nar
+
+## Kafka Connect based Connectors
+
+The following connectors are based on Kafka Connect protocol. [This documentation](https://github.com/datastax/pulsar-3rdparty-connector) explains more details about each connector.
```
Comment (mendonk) [inline Luna_Streaming_Experimental.md:30]:
```suggestion
The following connectors are based on the Kafka Connect protocol. [This documentation](https://github.com/datastax/pulsar-3rdparty-connector) provides more details about individual connectors.
```

**Review by mendonk — APPROVED** [review]

File: Luna_Streaming_Experimental.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+# DataStax Luna Streaming Experimental Distribution
```
Comment (eolivelli) [inline Luna_Streaming_Experimental.md:1]: this file must be versioned as well

**Review by eolivelli — CHANGES_REQUESTED** [review]

File: Luna_Streaming_Experimental.md (hunk 4 above)
Comment (nicoloboschi) [reply Luna_Streaming_Experimental.md]: I removed the new page
I updated the 2.10 1.1, see here https://github.com/datastax/release-notes/blob/master/Luna_Streaming_2.10_Release_Notes.md#luna-streaming-distribution-210-11

**Pull request merged**